rollup/sync_service.go

```diff
@@ -313,7 +313,6 @@ func (s *SyncService) VerifierLoop() {
 			if err != nil {
 				log.Error("Cannot apply transaction", "msg", err)
 			}
-			s.SetLatestIndex(&i)
```

Comment:
This can be removed because it now occurs within `applyTransaction` which would be called by `maybeApplyTransaction`. Is that correct?